src/wx/widgets/sdljoy.cpp

```diff
@@ -171,7 +172,7 @@ wxSDLJoyState::wxSDLJoyState(int sdl_index)
 
 wxSDLJoyState::wxSDLJoyState(wxJoystick joystick) : wx_joystick_(joystick) {
     int sdl_index = wx_joystick_.sdl_index_;
-    if (SDL_IsGameController(sdl_index)) {
+    if (gopts.sdl_game_controller_mode && SDL_IsGameController(sdl_index)) {
```

Comment:
Can you use `OPTION` instead here? I'm trying to get rid of `gopts` to create explicit ownership for everything.
My long term goal is to make it easier to have multiple emulator instances in separate threads to improve some of the Link functionality.